egs/chime6/asr1/local/prepare_dict.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
```

Comment:
Probably we don't need this?

Reply:
Yes, this file is redundant. We removed it and some other unused scripts.